src/cdk/v2/destinations/smartly/data/trackMapping.json

```diff
@@ -0,0 +1,69 @@
+[
+  {
+    "sourceKeys": "event",
+    "required": true,
+    "destKey": "event"
+  },
+  {
+    "destKey": "value",
+    "sourceKeys": [
+      "properties.total",
+      "properties.value",
+      "properties.revenue",
+      {
+        "operation": "multiplication",
+        "args": [
+          {
+            "sourceKeys": "properties.price"
+          },
+          {
+            "sourceKeys": "properties.quantity",
+            "default": 1
+          }
+        ]
+      }
+    ],
+    "metadata": {
+      "type": "toNumber"
+    },
+    "required": false
+  },
+  {
+    "sourceKeys": ["properties.conversions", "properties.products.length"],
+    "required": false,
+    "metadata": {
+      "defaultValue": "1"
+    },
+    "destKey": "conversions"
+  },
+  {
+    "sourceKeys": ["properties.adUnitId", "properties.ad_unit_id"],
+    "required": true,
+    "destKey": "ad_unit_id"
+  },
+  {
+    "sourceKeys": ["properties.platform"],
+    "required": true,
+    "destKey": "platform"
+  },
+  {
+    "sourceKeys": ["properties.adInteractionTime", "properties.ad_interaction_time"],
+    "required": true,
+    "destKey": "ad_interaction_time"
+  },
+  {
+    "sourceKeys": ["properties.installTime"],
+    "required": false,
+    "destKey": "installTime"
+  },
+  {
+    "sourceKeys": ["originalTimestamp", "timestamp"],
+    "required": false,
+    "destKey": "event_time"
```

Comment:
Typically "originalTimestamp", "timestamp" will come in UTC format, but based on research spec smartly will expect it in UNIX format. Where were are handling that conversion of units.